packages/api/src/command/medical/document/check-doc-queries.ts

```diff
@@ -116,52 +116,53 @@ export async function updateDocQueryStatus(patients: PatientsWithValidationResul
   );
 }
 
-async function updatePatientsInSequence(patients: [string, GroupedValidationResult][]) {
-  for (const [patientId, { cxId, ...whatToUpdate }] of patients) {
-    await executeOnDBTx(PatientModel.prototype, async transaction => {
-      const patientFilter = {
-        id: patientId,
-        cxId: cxId,
-      };
-      const patient = await getPatientOrFail({
-        ...patientFilter,
-        transaction,
-      });
-      const docProgress = patient.data.documentQueryProgress;
-      if (!docProgress) {
-        console.log(`Patient without doc query progress @ update, skipping it: ${patient.id} `);
-        return;
-      }
-      if (whatToUpdate.convert) {
-        const convert = docProgress.convert;
-        docProgress.convert = convert
-          ? {
-              ...convert,
-              status: getStatus(convert),
-              total: calculateTotal(convert),
-            }
-          : undefined;
-      }
-      if (whatToUpdate.download) {
-        const download = docProgress.download;
-        docProgress.download = download
-          ? {
-              ...download,
-              status: getStatus(download),
-              total: calculateTotal(download),
-            }
-          : undefined;
-      }
-      const updatedPatient = {
-        ...patient,
-        data: {
-          ...patient.data,
-          documentQueryProgress: docProgress,
-        },
-      };
-      await PatientModel.update(updatedPatient, { where: patientFilter, transaction });
+async function updatePatientsInSequence([patientId, { cxId, ...whatToUpdate }]: [
```

Comment:
just removed the loop, now we only have to provide a function to execute one item of the array